rustls/src/server/builder.rs

```diff
@@ -88,6 +88,33 @@ impl ConfigBuilder<ServerConfig, WantsServerCert> {
         Ok(self.with_cert_resolver(Arc::new(resolver)))
     }
 
+    /// Set a single raw public key certificate and matching private key. This
```

Comment:
I think I touched on this in previous comments, but is it appropriate to call this a "raw public key certificate"? I think "certificate" is fairly well established to mean the X.509v3 structure from RFC5280. I think a raw public key is a different beast and we should try not to conflate it with a certificate.